<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageThird</class>
 <widget class="QWidget" name="pageThird">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="bxcwidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="bxcframe">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QColumnView" name="columnView">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
